src/utils/generator.js

```diff
@@ -327,7 +310,10 @@ export class Generator {
     for (let entry of this.body) {
       if (!entry.isPure || !entry.declared || !context.canOmit(entry.declared)) {
         let nodes = entry.args.map((boundArg, i) => context.serializeValue(boundArg));
-        context.emit(entry.buildNode(nodes, context));
+        const buildNodeResult = entry.buildNode(nodes, context);
+        if (buildNodeResult) {
```

Comment:
This new special case shouldn't be needed, and the Flow type of buildNode is just that this case is actually infeasible.

Reply:
Again, `buildNodeResult` will already be truthy, no need for this `if` statement.